system/websocket/0001-Porting-the-code-for-NuttX.patch

```diff
@@ -0,0 +1,2257 @@
+From be917069caddfc701169f165c6900ee3cc919f0e Mon Sep 17 00:00:00 2001
+From: SPRESENSE <41312067+SPRESENSE@users.noreply.github.com>
+Date: Fri, 19 Apr 2019 17:40:58 +0900
+Subject: [PATCH] Porting the code for NuttX
+
+---
+ cwebsocket/src/cwebsocket/client.c | 801 +++++++++++++++++++----------
+ cwebsocket/src/cwebsocket/client.h | 177 ++++++-
+ cwebsocket/src/cwebsocket/common.c |  29 +-
+ cwebsocket/src/cwebsocket/common.h |  68 ++-
+ cwebsocket/src/cwebsocket/scanw.c  | 512 ++++++++++++++++++
```

Comment:
look like we make a huge change to the original code, do we really so much change?

Reply:
for now is needed. later maybe we can reduce the patch even more